custom-networks.py

```diff
@@ -0,0 +1,179 @@
+import csv
```

Comment:
Like in device.py, can we add some header content (check lines 1-4 in device.py

Reply:
Fixed